CASC/PortfolioMode.cpp

```diff
@@ -134,8 +134,11 @@ bool PortfolioMode::searchForProof()
     //we normalize now so that we don't have to do it in every child Vampire
     ScopedLet<Statistics::ExecutionPhase> phaseLet(env.statistics->phase,Statistics::NORMALIZATION);
     Normalisation().normalise(*_prb);
-
-    TheoryFinder(_prb->units(),property).search();
+    
+    //TheoryFinder can not yet find polymorphic theory axioms
```

Comment:
Not being able to find something it was not meant to find does not seem like a reason not to call.

Does TheoryFinder cope with polymorphic input?

Reply:
No it doesn't, it crashes. I should probably have worded that comment differently.